src/h5web/vis-packs/core/shared/TooltipMesh.module.css

```diff
@@ -1,6 +1,6 @@
 .tooltip {
   padding: 0.25rem 0.5rem;
-  background-color: var(--h5w-tooltip--bgColor, whitesmoke);
+  background-color: var(--h5w-tooltip--bgColor, rgba(255, 255, 255, 0.9));
```

Comment:
Making the background of the tooltip slightly transparent like the domain slider tooltip.